index.js

```diff
@@ -34,10 +35,33 @@ CompassCompilerPlugin.prototype.toTree = function(tree, inputPath, outputPath, i
 
 
   var outputPaths = compassOptions.outputPaths;
-  var trees = Object.keys(outputPaths).reduce(function(trees, file) {
+  var trees = Object.keys(outputPaths).map(function(file) {
 
     // Watch inputTrees and compassOptions.importPath directories
-    var inputTrees = [inputPath];
+    var inputTrees = [tree];
+
+    // Define getSassDir function if not overridden by user
+    if (!compassOptions.getSassDir) {
+      compassOptions.getSassDir = function(inputTrees, inputPaths) {
```

Comment:
Yes. It's one of the reasons why I wanted to make the changes in the `broccoli-compass-compiler` in the beginning.